socorro/lib/ver_tools.py

```diff
@@ -106,12 +113,14 @@ def _convert_to_str(x):
 _denormalize_fn_list = (_convert_to_str, _convert_to_str,
                         _convert_to_str, _convert_to_str)
 
+
 class NotAVersionException(Exception):
     """An exception raised by the class VersionComparer if given a string
     outside the standards of a version string as specified by the regular
     expression _version_part_re"""
     pass
 
+
 @memoize(1000)
 def normalize(version_string, max_version_parts=4):
```

Comment:
consider pruning this or moving this method and deleting the rest. The SupportClassificationRule class has other normalize methods. Maybe it belongs in there.